pkg/apis/serving/v1alpha1/roundtrip_test.go

```diff
@@ -1,67 +0,0 @@
-/*
-Copyright 2020 The Knative Authors
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package v1alpha1
-
-import (
-	"testing"
-
-	"k8s.io/apimachinery/pkg/api/apitesting/fuzzer"
-	"k8s.io/apimachinery/pkg/runtime"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	pkgfuzzer "knative.dev/pkg/apis/testing/fuzzer"
-	"knative.dev/pkg/apis/testing/roundtrip"
-	v1 "knative.dev/serving/pkg/apis/serving/v1"
-	"knative.dev/serving/pkg/apis/serving/v1beta1"
-)
-
-func TestServingRoundTripTypesToJSON(t *testing.T) {
-	scheme := runtime.NewScheme()
-	utilruntime.Must(AddToScheme(scheme))
-
-	fuzzerFuncs := fuzzer.MergeFuzzerFuncs(
-		pkgfuzzer.Funcs,
-		v1.FuzzerFuncs,
-	)
-	roundtrip.ExternalTypesViaJSON(t, scheme, fuzzerFuncs)
-}
-
-func TestServingRoundTripTypesToAlphaHub(t *testing.T) {
```

Comment:
nit: `TestServingRoundTripTypesToAlphaHub` => `TestServingRoundTripTypesToBetaHub`

Reply:
I'm pretty sure I had you add a beta version when you added these, so I think it already exists under `v1beta1`.  checking...

Reply:
yup, it's there. `pkg/apis/serving/v1beta1/roundtrip_test.go`

Reply:
wooot